src/components/articles/viewArticles.js

```diff
@@ -0,0 +1,42 @@
+import React, { Component } from "react";
+import { connect } from "react-redux";
+import ViewArticles from "../../views/articles/viewArticles";
+// import { fetchOneOrAllArticlesAction } from "../../actions/articleActions/articleAction";
+// import articlesReducer from "../../reducers/articlesReducer/articleReducer";
+// import PropTypes from "prop-types";
+class ViewArticle extends Component {
+  constructor(props) {
+    super(props);
+    // const articles = this.props.articles;
+    this.state = {
+    //   articles
+    };
+  }
+
+  componentDidMount() {
+    // const { articles } = this.state;
+    // this.props.dispatch(fetchOneOrAllArticlesAction);
+  }
+
+  render() {
+    // console.log(this.props);
```

Comment:
Remove all the commented code and comments, the code will be added when this task is being worked on

Reply:
The viewArticle.js file has been removed since we aren't working with it yet